### ISSUE TITLE
Replace Team-CGKM to more appropriate name

### ISSUE THREAD

**Issue by KevinKelly25** [arcplanner/ArcPlanner#2]
As we have moved on from Team-CGKM, a team made for CS298 DevOps at Western Connecticut State University, I think it is appropriate to either think of a new team name or just plainly say ArcPlanner in place of Team-CGKM.

**Comment by MatthewChastain** [comment]
I agree that simply using *ArcPlanner* as the team name would be sufficient. If there are other ideas for a new team name other than that though I'd like to consider them.
